src/js/common/schemaform/FormApp.jsx

```diff
@@ -159,8 +169,7 @@ class FormApp extends React.Component {
     const isConfirmationPage = trimmedPathname.endsWith('confirmation');
     const GetFormHelp = formConfig.getHelp;
     let content;
-
-    if (!formConfig.disableSave && this.props.loadedStatus === LOAD_STATUSES.pending) {
+    if (!formConfig.disableSave && trimmedPathname.endsWith('resume')) {
```

Comment:
Is this right? We can still be loading a form and not on the `resume` page, I think (e.g. loading a saved form on the intro page).

Reply:
Yes, this should check for either of these conjunctive clauses. Thanks @jbalboni, will change.